staff/carlota-vicens/arrays/main.js

```diff
@@ -0,0 +1,19 @@
+//add all elements
```

Comment:
its better to use 'use strict';

Reply:
ho tindre en compte per la versio 2.0.0 @bcasasus